pkg/internal/job_status.go

```diff
@@ -7,60 +7,52 @@ import (
 )
 
 // GetScheduledJobStatus Gets job status
-func GetScheduledJobStatus(jobStatus radixv1.RadixBatchJobStatus, stopJob bool) (status radixv1.RadixBatchJobApiStatus) {
-	status = radixv1.RadixBatchJobApiStatusWaiting
-	switch jobStatus.Phase {
-	case radixv1.BatchJobPhaseActive:
-		status = radixv1.RadixBatchJobApiStatusActive
-	case radixv1.BatchJobPhaseRunning:
-		status = radixv1.RadixBatchJobApiStatusRunning
-	case radixv1.BatchJobPhaseSucceeded:
-		status = radixv1.RadixBatchJobApiStatusSucceeded
-	case radixv1.BatchJobPhaseFailed:
-		status = radixv1.RadixBatchJobApiStatusFailed
-	case radixv1.BatchJobPhaseStopped:
-		status = radixv1.RadixBatchJobApiStatusStopped
-	case radixv1.BatchJobPhaseWaiting:
-		status = radixv1.RadixBatchJobApiStatusWaiting
+func GetScheduledJobStatus(jobStatus radixv1.RadixBatchJobStatus, stopJob bool) string {
+	status := jobStatus.Phase
+	if len(status) == 0 {
+		status = radixv1.BatchJobPhaseWaiting
 	}
-	if stopJob && (status == radixv1.RadixBatchJobApiStatusWaiting || status == radixv1.RadixBatchJobApiStatusActive || status == radixv1.RadixBatchJobApiStatusRunning) {
-		return radixv1.RadixBatchJobApiStatusStopping
+	if stopJob && (status == radixv1.BatchJobPhaseWaiting || status == radixv1.BatchJobPhaseActive || status == radixv1.BatchJobPhaseRunning) {
+		return "Stopping"
```

Comment:
could we re-introduce enums for status variables?

Reply:
Used `radixv1.RadixBatchJobApiStatus` as return but keep `string` for `Status` field (I assume to change it can affect existing interface on user side)